containerd/pkg.yaml

```diff
@@ -15,7 +15,7 @@ steps:
       PKG_CONFIG_PATH: /usr/lib/pkgconfig
       CC: /toolchain/bin/gcc
       PATH: /toolchain/go/bin:/toolchain/bin:{{ .PATH }}
-      BUILDTAGS: 'seccomp no_aufs no_btrfs no_devmapper no_systemd no_zfs'
+      BUILDTAGS: 'seccomp no_aufs no_btrfs no_devmapper no_plugin no_systemd no_zfs'
```

Comment:
what does this do?

Reply:
please see https://github.com/containerd/containerd/pull/11213

Reply:
I messed up the tag name, but still there's something wrong there... it has no effect

Reply:
shouldn't it be `no_dynamic_plugins` ?

Reply:
yes, that's what I messed up, but I fixed it, thank you!